logger/src/commonMain/kotlin/com/wire/kalium/logger/KaliumLogger.kt

```diff
@@ -63,7 +63,7 @@ class KaliumLogger(config: Config, logWriter: LogWriter? = null) {
         } else {
             KermitLogger(
                 config = StaticConfig(
-                    minSeverity = config.severityLevel, listOf(logWriter, platformLogWriter())
+                    minSeverity = config.severityLevel, logWriterList
```

Comment:
Shouldn't we also be including the `platformLogWriter()` along with the values we receive on `logWriterList`?

Reply:
Not including the `platformLogWriter()` is reason why I did this change. I don't want to log to the system.out when running the CLI app which is what the `platformLogWriter()` do on the JVM. This is the breaking change where if you add any custom `LogWriter` you now also need to include the `platformLogWriter()` yourself.

I would open a PR on android reloaded to fix this.

Reply:
Ooooh nice then! Thanks!